main.go

```diff
@@ -1,39 +1,52 @@
 package main // import "moul.io/moulsay"
 
 import (
+	"context"
+	"flag"
 	"fmt"
 	"io/ioutil"
 	"log"
 	"os"
 	"strings"
 
-	cli "gopkg.in/urfave/cli.v2"
+	"github.com/peterbourgon/ff/v3/ffcli"
 	"moul.io/moulsay/moulsay"
 )
 
 func main() {
-	app := cli.App{
-		Flags: []cli.Flag{
-			&cli.IntFlag{Name: "max-width", Aliases: []string{"w"}, Value: 72},
+	fs := flag.NewFlagSet("app", flag.ExitOnError)
+	maxWidth := fs.Int("max-width", 72, "set max width")
+
+	root := &ffcli.Command{
+		Name:       "moul say",
+		ShortUsage: "moulsay + word",
+		ShortHelp:  "moulsay word",
+		LongHelp:   "moulsay -max-width=? word for different max width from moul to word (minimum max-width = 27)",
+		FlagSet:    fs,
+		Exec: func(ctx context.Context, args []string) error {
+			err := run(ctx, args, *maxWidth)
+			if err != nil {
+				return err
+			}
+			return nil
```

Comment:
```suggestion
			return run(ctx, args, *maxWidth)
```

if you don't make any wrapping, please use this short form instead